HACKING.md

```diff
@@ -273,17 +273,38 @@ follows:
 
 ### [PROD] Create quay.io Fedora image push secret
 
-This secret is used to push the oscontainer and others to Quay.io
+This secret is used to push the oscontainer and others to Quay.io.
+The secret can be obtained from the `fedora-push-registry-secret` in BitWarden.
 
-1. Obtain the file `fedora-push-registry-secret` from BitWarden.
-2. Run: `$ oc create secret generic fedora-push-registry-secret --from-file=dockercfg=fedora-push-registry-secret`.
+After obtaining the secret data you can create the kubernetes secret via:
+
+```
+oc create secret generic fedora-push-registry-secret \
+    --from-literal=filename=dockercfg \
+    --from-file=data=fedora-push-registry-secret
+oc label secret/fedora-push-registry-secret \
+    jenkins.io/credentials-type=secretFile
+oc annotate secret/fedora-push-registry-secret \
+    jenkins.io/credentials-description="Registry push secret for pushing the CoreOS OSContainer"
```

Comment:
Minor: feels like "Registry push secret for CoreOS OSContainer" would be sufficient? (And similarly below.)